_posts/2018-06-04-baring-it-all-in-your-whitepaper.md

```diff
@@ -0,0 +1,41 @@
+
+Two approaches that are open and provide clarity without digging yourself a hole to dragged into later on in the process.
+
+## About giving up your seekrits!
+
+In terms of product features etc - I tend to take a slightly different view to many others, so caveat emptor on the following ramblings from me.
+
+I don't believe in competitors, there may be other companies that validate my target market, or create disenfranchised customers for me to pick up. They are both great things. Ultimately success that I've seen has been more based on the support of those who believe and connect with the world view of your product. That comes back to some very lean principles; get out of the building, connect, learn, fail, then produce something that resonates with your customers. There's always going to be people creating similar products.
+
+I see it like offices, you walk into a business and make a judgement about how you feel about them based on the way they decorate their office. Some you will connect with, others less so. We can't build the product that satisfies everyone, so make your "office" awesome for you and those close to you, that passion will translate into awesome relationships with your customers and a product experience they connect with.
+
+Now the more blockchain related perspective: TSEs that are most successful are the ones that understand that they are creating something that will ultimately be controlled by the community. The products that fit into this view need to be understood by the community because protocols are only useful if anyone can implement them. If you're running a TSE for a product that will never play nice the community then you're most likely either: a) not being ambitious enough or b) probably not an ideal candidate for running a TSE. Hopefully you're just in (a) because that just means thinking bigger!
+
+If you can think about how your solution will benefit the community, even if you operate it with full control in the early days, you're well onto your way to articulating something that investors will love to see in your whitepaper. It's easy to be open about something that is going to be amazing for a whole community of people. Investors will be happy to accept that you will maintain control initially if they can see the bigger picture.
+
+I'm more worried about selling my product once for the first time than I am about someone else trying to do it too. If you bare all (or something close to it) in a TSE you are going to be able to prove you did it first. Everyone else will copy it one day anyway. Also you will be well funded and have a great community ready to go with the product when it's live!
+
```

Comment:
I suggest you close with a summary of what we encourage in this space and how the ES platform provides quality standards and guidelines.  Obviously not a hard sell, more along the lines of haring what you are working on and believe in.

Reply:
Good idea, thanks